src/mut/morphology.cpp

```diff
@@ -357,12 +354,14 @@ void Morphology::write(const std::string& filename) {
         extension += my_tolower(c);
 
     if (extension == ".h5")
-        writer::h5(clean, filename);
+        writer::h5(*this, filename);
     else if (extension == ".asc")
-        writer::asc(clean, filename);
-    else if (extension == ".swc")
+        writer::asc(*this, filename);
+    else if (extension == ".swc") {
+        morphio::mut::Morphology clean(*this);
+        clean.sanitize();
```

Comment:
Can you add a comment here on why we must sanitize only for swc ?

Reply:
I think it would make sense to, throw an exception when a format can't represent the data, and in the exception say how to fix it - that way ppl know they have a problem, and that they need to solve it.